### PR TITLE
feat: build Docker images for arm64

### PR DESCRIPTION
**Pull request by mikejgray**
# Description
Adds a section to the Docker build action specifying that it should build both AMD64 and ARM64 images.

# Issues
N/A

# Other Notes
N/A

### CODE REVIEW

File: .github/workflows/publish_docker.yml [hunk 1]
```diff
@@ -61,6 +61,7 @@ jobs:
         with:
           context: .
           push: true
+          platforms: linux/amd64,linux/arm64
```
Comment (NeonDaniel) [inline .github/workflows/publish_docker.yml:64]: Could this be moved to a setting, similar to [registry](https://github.com/NeonGeckoCom/.github/blob/master/.github/workflows/publish_docker.yml#L11-L13) so we can specify non-default platforms?

**Review by NeonDaniel — COMMENTED** [review]
Not sure if this is ready for review yet, but I left one comment

File: .github/workflows/publish_docker.yml (hunk 1 above)
Comment (mikejgray) [reply .github/workflows/publish_docker.yml]: Good suggestion! Incorporated.

**Comment by NeonDaniel** [comment]
This automation is failing https://github.com/NeonGeckoCom/neon_messagebus/actions/runs/4546321377/jobs/8014830968
Possibly related discussion: https://github.com/docker/build-push-action/issues/302

**Comment by NeonDaniel** [comment]
Tested with neon-messagebus https://github.com/NeonGeckoCom/neon_messagebus/pkgs/container/neon_messagebus

I see the OS/arch is listed as 'unknown', can you make sure the image is properly resolved when you pull from an arm device?

**Comment by mikejgray** [comment]
@NeonDaniel It didn't work, I'll do some troubleshooting and testing of the action and let you know...

```shell
❯ docker pull ghcr.io/neongeckocom/neon_messagebus:0.3.2-a.1
0.3.2-a.1: Pulling from neongeckocom/neon_messagebus
no matching manifest for linux/arm64/v8 in the manifest list entries
❯ docker run ghcr.io/neongeckocom/neon_messagebus:0.3.2-a.1
Unable to find image 'ghcr.io/neongeckocom/neon_messagebus:0.3.2-a.1' locally
0.3.2-a.1: Pulling from neongeckocom/neon_messagebus
docker: no matching manifest for linux/arm64/v8 in the manifest list entries.
See 'docker run --help'.
❯ docker pull ghcr.io/neongeckocom/neon_messagebus:0.3.2-a.1@sha256:6c0a420dd416deea201b6734cff1ba23506949c6a859fd2f2c2af73a19c10f55
ghcr.io/neongeckocom/neon_messagebus@sha256:6c0a420dd416deea201b6734cff1ba23506949c6a859fd2f2c2af73a19c10f55: Pulling from neongeckocom/neon_messagebus
unsupported media type application/vnd.in-toto+json
```

**Comment by NeonDaniel** [comment]
> @NeonDaniel It didn't work, I'll do some troubleshooting and testing of the action and let you know...

Maybe it needs `linux/arm64/v8` instead of `linux/arm64`? I haven't looked but I suspect some packages on Dockerhub could be checked for common/valid arch specs

**Comment by mikejgray** [comment]
Closing in favor of #62 (I forgot this was still open) :)

**Pull request closed without merging**